dev-utils/webdriver.js

```diff
@@ -155,7 +159,7 @@ function getWebdriveBaseConfig(
     ...capability
   }))
 
-  return {
+  const baseConfig = {
```

Comment:
The `services` config still has `sauce` and other saucelabs specific options in it, doesn't those interfere with the local execution? What do you think about refactoring the configs object to separate the saucelabs related options?

Reply:
All those options would be dropped by `webdriver` when the `automationprotocol` is set to `devtools` so thats why I didnt spend time thinking which configs should be similar.